ztp/siteconfig-generator/siteConfig/testdata/filteredoutput/onlyUserCR.yaml

```diff
@@ -116,6 +116,7 @@ metadata:
     annotations:
         argocd.argoproj.io/sync-wave: "1"
         ran.openshift.io/ztp-gitops-generated: '{}'
+        ran.openshift.io/ztp-warning-field-deprecation-extraManifestPath: extraManifestPath will be deprecated after OCP 4.15, please use ExtraManifests.SearchPaths for OCP versions >= 4.14
```

Comment:
Shorter annotation key is better something like "ran.openshift.io/field-deprecation"

Reply:
for the cpuset deprecation the annotation is formed like below and i followed to be consistent:

`ran.openshift.io/ztp-warning-field-deprecation-cpuset`